x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/AbstractEnrichProcessor.java

```diff
@@ -155,13 +171,12 @@ int getMaxMatches() {
 
     private static BiConsumer<SearchRequest, BiConsumer<SearchResponse, Exception>> createSearchRunner(Client client) {
         return (req, handler) -> {
-            client.execute(EnrichCoordinatorProxyAction.INSTANCE, req, ActionListener.wrap(
-                resp -> {
-                    handler.accept(resp, null);
-                },
-                e -> {
-                    handler.accept(null, e);
-                }));
+            client
+                .execute(
```

Comment:
I'm not so happy that `.execute(` is on a new line. This happens also a lot in other places.

Reply:
I've gone backwards and forwards on this setting ("Qualified invocations") a few times. I think we should actually change it back to what I originally used, which was "Wrap all elements, except first element if not necessary". It doesn't work in every situation, but it's probably better on balance.

You can change this setting with the following patch. Would you mind applying it?

```patch
diff --git a/.eclipseformat.xml b/.eclipseformat.xml
index 9ff0aa1f408..ebda0af1503 100644
--- a/.eclipseformat.xml
+++ b/.eclipseformat.xml
@@ -205,7 +205,7 @@
         <setting id="org.eclipse.jdt.core.formatter.alignment_for_resources_in_try" value="48"/>
         <setting id="org.eclipse.jdt.core.formatter.use_tabs_only_for_leading_indentations" value="false"/>
         <setting id="org.eclipse.jdt.core.formatter.parentheses_positions_in_try_clause" value="separate_lines_if_wrapped"/>
-        <setting id="org.eclipse.jdt.core.formatter.alignment_for_selector_in_method_invocation" value="48"/>
+        <setting id="org.eclipse.jdt.core.formatter.alignment_for_selector_in_method_invocation" value="80"/>
         <setting id="org.eclipse.jdt.core.formatter.never_indent_block_comments_on_first_column" value="false"/>
         <setting id="org.eclipse.jdt.core.formatter.keep_code_block_on_one_line" value="one_line_if_empty"/>
         <setting id="org.eclipse.jdt.core.formatter.insert_space_after_opening_paren_in_synchronized" value="do not insert"/>
```

Reply:
Thanks, I've changed that setting via: 910a5ba9f657b3c9a9a9cefde4729fb462dce2a9 
This is looks better now